Code/GraphMol/MolAlign/AlignMolecules.cpp

```diff
@@ -30,7 +30,7 @@ double getAlignmentTransform(const ROMol &prbMol, const ROMol &refMol,
   if (atomMap == 0) {
     // we have to figure out the mapping between the two molecule
     MatchVectType match;
-    if (SubstructMatch(refMol, prbMol, match)) {
+    if (SubstructMatch(refMol, prbMol, match, true, false, true)) {
```

Comment:
In general I don't like magic numbers (or magic booleans in this case).  It makes it easier on future devs to be explicit and also helps validate the pull request :)

```
const bool RecursionPossible=true;
const bool UseChirality=false;
const bool UseQueryQueryMatches=true;
if (SubstructMatch(ref mol, prbMol, match, 
                              RecursionPossible, UseChirality, UseQueryQueryMatches))
```